frontend/src/components/funding/FundingAgreementsTab.vue

```diff
@@ -92,7 +92,7 @@ export default {
             applications?.map((application) =>
               ApplicationService.getSupplementaryApplicationsByDate(
                 application.applicationId,
-                `statusCode=${SUPPLEMENTARY_APPLICATION_STATUS_CODES.APPROVED}`,
+                `statusCode=${SUPPLEMENTARY_APPLICATION_STATUS_CODES.APPROVED}&pcmValidated=true`,
```

Comment:
I know it's existing code but I'm not a fan of passing in a generic filterQuery in our services as it allows the caller to pass in absolutely anything. Since it's a pretty minor refactor can we just pass in 4 explicit params instead (status, startDateFrom, startDateTo, pcmValidated)?

Reply:
done!